go.mod

```diff
@@ -192,7 +192,6 @@ replace (
 	// Ref: https://forum.cosmos.network/t/ibc-security-advisory-dragonberry/7702
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
 	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.46.7
-	github.com/ethereum/go-ethereum => github.com/crypto-org-chain/go-ethereum v1.10.19-deepcopy-jumptable
```

Comment:
seems like `ethermint v0.20.2-cronos` is not compatible with `go-ethereum v1.10.26`

@yihuang is the replace still required ?

Reply:
not required, but if ethermint is not compatible, then let's do the migration in another PR, can keep it for now.